pkg/cmd/pipeline.go

```diff
@@ -0,0 +1,43 @@
+// Copyright © 2019 The Knative Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+)
+
+var (
+	namespace string
+)
+
+// pipelinesCmd represents the pipelines command
+var pipelinesCmd = &cobra.Command{
+	Use:     "pipelines",
+	Aliases: []string{"p", "pipeline"},
+
+	Short: "Handles pipelines",
+	Long:  `add long description`, // TODO(sthaha)
```

Comment:
Let's add something like `Manage pipelines` for now :wink:

Reply:
> Few comments, and one question : do we want to vendor files or not ? (`go mod vendor`)

Thanks for the review. 
Is my understanding wrong that `go.sum` would ensure that all of us use the same version of dependencies? I am happy to add `vendor` if you think it is better.

Reply:
> Is my understanding wrong that `go.sum` would ensure that all of us use the same version of dependencies? 

This is correct — the only benefit to add vendor are :
- if something goes down you still have the code (this one is a bit *meh* :sweat_smile:)
- you don't need to download the deps to build the binary (less *meh*) and you can build the binary with older version of go (if it make sense) and anywhere (aka in `GOPATH` without having to set `GO111MODULE` env..)

Reply:
ack. I will add it in